.github/workflows/e2e-test-k3s.yml

```diff
@@ -5,17 +5,20 @@
 #    - Checkout the repository code.
 #    - Cache the ~/.cache/go-build and ~/go/pkg/mod directories to improve performance.
 #    - Set up a docker registry and k3s cluster using a script called k3s-setup.sh.
-#    - Set the pull request number as an environment variable.
-#    - Run Go tests located in the testing/go directory, ensuring that the correct version of Go is used and the Go modules are tidied up before the tests are run.
-#
-#  Overall, this action ensures that the code changes made in the pull request do not break any end-to-end tests on a k3s cluster.
+#    - Set the pull request number, OCI module image name as an environment variables.
+#    - Wait until the registry contains OCI module image for current PR (pushed by prow-job)
+#    - Run test script located in the testing directory
 
+name: Run E2E tests on k3s cluster
```

Comment:
Could we run this test on main too? I mean, when we release will be nice to check if all tests are green.

Reply:
Script is ready for this, I just need to test such scenario.